core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -882,12 +879,7 @@ public synchronized void stopInternal() throws InterruptedException, IOException
    */
   public synchronized CompletableFuture<RaftClientReply> sendMessageAsync(
       RaftPeerId server, Message message) {
-    long timeoutMs = ServerConfiguration.getMs(
-        PropertyKey.MASTER_EMBEDDED_JOURNAL_SNAPSHOT_DATA_REQUEST_TIMEOUT);
-    long retryMs = ServerConfiguration.getMs(
-        PropertyKey.MASTER_EMBEDDED_JOURNAL_SNAPSHOT_DATA_REQUEST_INTERVAL);
-    RaftClient client = createClient(TimeDuration.valueOf(timeoutMs, TimeUnit.MILLISECONDS),
-        RetryPolicies.retryForeverWithSleep(TimeDuration.valueOf(retryMs, TimeUnit.MILLISECONDS)));
+    RaftClient client = createClient();
```

Comment:
I know it was the behavior before but should we consider cleaning up these clients?

Reply:
That would definitely be worthwhile, maybe that would be best suited for a followup PR?

Reply:
Sure. Let's just make sure to not forget and treat the subject sometime soon. (a ticket would suffice at this point.)

Reply:
Follow-up: https://github.com/Alluxio/alluxio/pull/15058